src/ring_theory/dedekind_domain.lean

```diff
@@ -108,33 +108,150 @@ structure is_dedekind_domain_dvr : Prop :=
 (is_dvr_at_nonzero_prime : ∀ P ≠ (⊥ : ideal A), P.is_prime →
   discrete_valuation_ring (localization.at_prime P))
 
+section inverse
+
+open_locale classical
+
+variables {R₁ : Type*} [integral_domain R₁] {g : fraction_map R₁ K}
+
+noncomputable instance fractional_ideal_has_div :
+  has_div (fractional_ideal g) :=
+⟨ λ I J, if h : J = 0 then 0 else ⟨I.1 / J.1, fractional_ideal.fractional_div_of_nonzero h⟩ ⟩
```

Comment:
We can delete this instance now that the `div_eq_mul_inv` refactor has been merged, right?

```suggestion
```

Reply:
Well, I think so. Is it wise to first comment it to see if everything compiles before deleting it?

Reply:
With `git revert` we can undo the commit that deletes it, so I almost never comment something out.